governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Onboard GNO to Spark Protocol - May 15, 2023
+summary: Onboard GNO (Gnosis) to Spark Protocol with included parameters. Reduce Maker GNO-A debt ceiling to zero.
```

Comment:
```suggestion
summary: Signal your support or opposition to onboarding GNO (Gnosis) to Spark Protocol with the following parameters.
```

Reply:
Happy to also leave it as before if the DC change should feature in the summary.

Reply:
Yep, change here was intentional.